### PR TITLE
Separating command and output for oc CLI docs

### PR DESCRIPTION
**Pull request by bergerhoffer**

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@openshift/team-documentation For peer review please

File: modules/cli-developer-application-management.adoc [hunk 1]
```diff
@@ -165,6 +185,7 @@ $ oc serviceaccounts get-token default
 Configure existing application resources.
 
 .Example: Sets the name of a secret on a BuildConfig
```
Comment (jeana-redhat) [inline modules/cli-developer-application-management.adoc:187]: Happened to notice this example title is not in the imperative like the rest.

**Review by jeana-redhat — COMMENTED** [review]
Changes LGTM, noticed one existing inconsistency in modules/cli-developer-application-management.adoc

**Pull request merged**